scripts/env.list

```diff
@@ -1,30 +1,6 @@
-PCS_IOTHUB_CONNSTRING
-PCS_STORAGEADAPTER_DOCUMENTDB_CONNSTRING
-PCS_TELEMETRY_DOCUMENTDB_CONNSTRING
-PCS_STREAMANALYTICS_DOCUMENTDB_CONNSTRING
-PCS_TELEMETRYAGENT_DOCUMENTDB_CONNSTRING
-PCS_IOTHUBREACT_ACCESS_CONNSTRING
-PCS_IOTHUBREACT_HUB_NAME
-PCS_IOTHUBREACT_HUB_ENDPOINT
-PCS_IOTHUBREACT_HUB_PARTITIONS
-PCS_IOTHUBREACT_AZUREBLOB_ACCOUNT
-PCS_IOTHUBREACT_AZUREBLOB_KEY
-PCS_IOTHUBREACT_AZUREBLOB_ENDPOINT_SUFFIX
-PCS_ASA_DATA_AZUREBLOB_ACCOUNT
-PCS_ASA_DATA_AZUREBLOB_KEY
-PCS_ASA_DATA_AZUREBLOB_ENDPOINT_SUFFIX
-PCS_EVENTHUB_CONNSTRING
-PCS_EVENTHUB_NAME
-PCS_TELEMETRY_STORAGE_TYPE
-PCS_AAD_TENANT
+PCS_KEYVAULT_NAME
 PCS_AAD_APPID
 PCS_AAD_APPSECRET
-PCS_TSI_FQDN
-PCS_AZUREBLOB_CONNSTRING
-PCS_ACTION_EVENTHUB_CONNSTRING
-PCS_ACTION_EVENTHUB_NAME
-PCS_LOGICAPP_ENDPOINT_URL
-PCS_SEED_TEMPLATE
 
 # URLs
 PCS_AUTH_WEBSERVICE_URL=http://127.0.0.1:9001/v1
```

Comment:
This and a bunch of the other variables below wouldn't be used right?

Reply:
Yes